dev/up

```diff
@@ -8,7 +8,7 @@ if ! which golangci-lint &>/dev/null; then brew install golangci-lint; fi
 if ! which shellcheck &>/dev/null; then brew install shellcheck; fi
 if ! which protoc &>/dev/null; then brew install protobuf; fi
 if ! which protoc-gen-go &>/dev/null; then go install google.golang.org/protobuf/cmd/protoc-gen-go@latest; fi
-if ! which mockgen &>/dev/null; then go install go.uber.org/mock/mockgen@latest; fi
+go install go.uber.org/mock/mockgen@latest
```

Comment:
Is there no way to tell if you have the correct mockgen and only run this if you have the wrong one?

Reply:
Okay, I found a hack to only run this if the version is not 0.4.0. Basically we can pin the uber mockgen on the current version of v0.4.0 and update as needed. Should be good indefinitely because it is running behind the golang one, which is on v1.6.0

Reply:
Nice work!